src/microbe_stage/MicrobeHUD.cs

```diff
@@ -477,6 +477,13 @@ private void UpdateMulticellularButton(Microbe player)
 
         multicellularButton.Visible = true;
 
+        // Show the become multicellular tutorial
```

Comment:
Or is the tutorial about just collecting more colony members? My intention with the issue about the tutorial was that the players can quite easily see the numbers go up on the label on the button as they gather more cells, so that doesn't need explaining really. Instead how to actually trigger the move to multicellular is the part that needs a tutorial. Though I guess it would be worth it to also have a tutorial about how to collect colony members, *but* that tutorial should appear if the player has binding agents and not when they've already managed to form a colony.